server/planning/commands/flag_expired_items_test.py

```diff
@@ -170,25 +173,24 @@ async def test_planning(self):
                     },
                 ],
             )
-            FlagExpiredItems().run()
-
-            self.assertExpired(
+            await flag_expired_items_handler()
+            await self.assertExpired(
                 "planning",
                 {
                     "p1": False,
                     "p2": False,
                     "p3": False,
                     "p4": False,
                     "p5": True,
-                    "p6": False,
+                    "p6": True,
```

Comment:
I'm wondering, why were `p6` and `p8` changed in the tests from `False` to `True`. Is there any logic or query that is not the same as the old code?